app/models/concerns/devise_setup.rb

```diff
@@ -0,0 +1,29 @@
+module DeviseSetup
+  require 'hours'
+
+  extend ActiveSupport::Concern
+
+  included do
+    if ActiveRecord::Base.connection.table_exists? 'users'
+      if User.count == 0
+        devise :database_authenticatable,
+               :recoverable,
+               :rememberable,
+               :trackable,
+               :validatable,
+               :confirmable,
+               :registerable,
+               :invitable
+      else
+        devise :database_authenticatable,
+               :recoverable,
+               :rememberable,
+               :trackable,
+               :validatable,
+               :confirmable,
+               :invitable
+      end
```

Comment:
I get that, I don't like it to create a default user. By changing the accounts/new form we also have to adjust the controller and the model of accounts to check on the single tenant mode